src/features/chart/components/Chart.tsx

```diff
@@ -25,9 +26,9 @@ const tenors = [
   { value: "max", label: "MAX" }
 ];
 
-const yaxisFormat = item => item.toFixed(2);
+const yaxisFormat = (item: any) => item.toFixed(2);
```

Comment:
Try not to `any` type wherever possible. Looking at the usage, it looks like `item` should be a `number`.